src/Evolution/Systems/Ccz4/TimeDerivative.cpp

```diff
@@ -310,12 +310,8 @@ void TimeDerivative<Dim>::apply(
       symmetrized_d_field_b,
       0.5 * (d_field_b(ti::k, ti::j, ti::I) + d_field_b(ti::j, ti::k, ti::I)));
 
-  for (size_t k = 0; k < Dim; k++) {
-    contracted_symmetrized_d_field_b->get(k) = d_field_b.get(k, 0, 0);
-    for (size_t i = 1; i < Dim; i++) {
-      contracted_symmetrized_d_field_b->get(k) += d_field_b.get(k, i, i);
-    }
-  }
+  ::tenex::evaluate<ti::k>(contracted_symmetrized_d_field_b,
+                           d_field_b(ti::k, ti::i, ti::I));
```

Comment:
@nilsdeppe I didn't notice this until now but I think that while this calculation isn't analytically wrong, it's not exactly accurate to the paper because it's not symmetrized here in the code. This value `contracted_symmetrized_d_field_b` is supposed to represent the contraction of the 3rd definition in eq 28 [here](https://arxiv.org/pdf/1707.09910.pdf). This contraction of the *symmetrized* value is seen in eq 12h, 12l, and 12m and that's where it is used in this time derivative computation.

I think the accurate TE would be:
```
::TensorExpressions::evaluate<ti_k>(
      contracted_symmetrized_d_field_b,
      0.5 * (d_field_b(ti_k, ti_l, ti_L) + d_field_b(ti_l, ti_k, ti_L)));
```
which includes the symmetrization.

Let me know your thoughts on this. Sorry for not catching this sooner!

Reply:
It should be what's in the paper

Reply:
I'll swap this in and update the commit message